src/styles/_carousel.scss

```diff
@@ -0,0 +1,50 @@
+.slider {
+	position: relative;
+	width: 100%;
+	margin: 0 auto;
+	height: 500px;
+	overflow: hidden;
+	white-space: nowrap;
+
+	&__wrapper {
+		position: relative;
+		height: 100%;
+		width: 100%;
+	}
+}
+
+.slide {
+	display: inline-block;
+	height: 100%;
+	width: 100%;
+}
+
+.arrow {
+	height: 50px;
+	width: 50px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	background: #f9f9f9;
+	border-radius: 50%;
+	cursor: pointer;
+	transition: transform ease-in .1s;
+	position: absolute;
+	top: 45%;
+	z-index: 100;
+	background-color: rgba(255, 255, 255, 0.5);
+	border-radius: 100%;
+
+	&--next {
+		right: 20px;
+	}
+	&--back {
+		left: 20px;
+	}
+}
+
+.fa-chevron-right:before,
+.fa-chevron-left:before {
+	color: #fff;
+	opacity: 1;
```

Comment:
You can remove this opacity. Is there any reason you're using opacity on this one?

Reply:
left over testing opacity inheritance from parent, this has been removed on my latest commit